subsys/testsuite/ztest/include/syscalls/log_msg2.h

```diff
@@ -0,0 +1,3 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Is this temporary until a proper solution is found or is this the "solution"?

Reply:
This is 'the solution'.

Reply:
yeap, I think this is proper solution, from what I understand ztest requires stubs for autogenerated headers